css/style.css

```diff
@@ -9,400 +14,2053 @@ body {
+  line-height: 16px;
+  letter-spacing: 1px;
+  opacity: 0.5;
+  position: absolute;
+  width: 220px;
+  height: 17px;
+  left: 55px;
+  top: 85px;
 }
 
-.bag-text{
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+.bag-text {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: 0.5;
+  position: absolute;
+  width: 368px;
+  height: 148px;
+  left: 55px;
+  top: 179px;
+  color: #000;
 }
 
 .product {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 16px; 
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
 }
 
 .bag-title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  height: 41px;
+  margin-left: 47px;
+  top: 34px;
+  width: 90%;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 1px;
+  opacity: 0.5;
+  position: absolute;
+  height: 17px;
+  margin: 0 0 0 47px;
+  top: 92px;
 }
 
 .total-look__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: 0.5;
+  position: absolute;
+  width: 75%;
+  height: 101px;
+  left: 50px;
+  top: 144px;
+  margin: 0;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
+  position: absolute;
+  height: 15px;
+  left: 1525px;
+  top: 2663px;
+  width: 115px;
+}
+
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #fae8e4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
+}
+
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
+
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px;
+}
+
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #fff;
+  min-width: 350px;
+}
+
+.bag__color {
+  position: absolute;
+  top: 375px;
+  left: 55px;
+}
+
+.color-wight,
+.color-blue,
+.color-green {
+  width: 30.18px;
+  height: 30px;
+  border-radius: 50%;
+  border: none;
+}
+
+.color-wight:hover,
+.color-blue:hover,
+.color-green:hover {
+opacity: 0.5;
+}
+
+.color-pink {
+  width: 38px;
+  height: 38px;
+  border-radius: 50%;
+  background: #f29f9f;
+  border: 1px solid #f29f9f;
+}
+
+.color-pink:hover {
+  opacity: 0.5;
+}
+
+.color-wight {
+  background: #ded;
+}
+
+.color-blue {
+  background: #a8bff9;
+}
+
+.color-green {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".color-green" to come before selector ".color-green:hover" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)